model_compression_toolkit/gptq/keras/quantizer/quantization_builder.py

```diff
@@ -70,12 +73,13 @@ def quantization_builder(n: common.BaseNode,
 
         quant_method = n.final_activation_quantization_cfg.activation_quantization_method
 
-        quantizer_class = get_inferable_quantizer_class(quant_target=QuantizationTarget.Activation,
+        quantizer_class = get_trainable_quantizer_class(quant_target=QuantizationTarget.Activation,
+                                                        quantizer_id=TrainingMethod.STE,
                                                         quant_method=quant_method,
-                                                        quantizer_base_class=BaseKerasInferableQuantizer)
+                                                        quantizer_base_class=BaseKerasActivationTrainableQuantizer)
+        cfg = get_trainable_quantizer_activation_config(n, None)
 
-        kwargs = get_inferable_quantizer_kwargs(n.final_activation_quantization_cfg, QuantizationTarget.Activation)
-
-        activation_quantizers.append(quantizer_class(**kwargs))
+        # freeze_quant_params is True since in GPTQ the quantization parameters should not be trained (previously, we used inferable quantizers).
```

Comment:
"the **activation** quantization parameters should not..."
and you can remove the "previously.." part IMO